pytest_tests/testsuites/contract/test_contract/deploy/neo-go.yml

```diff
@@ -0,0 +1,11 @@
+name: testctr
+sourceurl: http://example.com/
+safemethods: []
+supportedstandards: []
+events:
+    - name: Hello world!
```

Comment:
Can be removed, there are no events in the code.

Reply:
fixed